ArrayUtils.js

```diff
@@ -1,21 +1,94 @@
 class ArrayUtils{
 
-    isEmpty(array){return false;}
+    isEmpty(array){
+        if(array[0]) {
+            return false;
+        } else {
+        return true;
+        }
+    }
 
-    append(original, value){return original;}
+    append(original, value){
+        original.push(value);
+        return original;
+    }
 
-    clone(original){return original;}
+    clone(original){
+        let copy = original;
+        return copy;
+    }
 
-    subArray(original, from, to){return original;}
+    subArray(original, from, to){
+        let newArray = [];
+        for(let i = from; i <= to; i++) {
+            newArray.push(original[i]);
+        }
+        return newArray;
+    }
 
-    equals(arr1, arr2){return false;}
+    equals(arr1, arr2){
+        let mismatch = 0;
+        if(arr1.length > arr2.length) {
+            for(let i = 0; i < arr1.length; i++) {
+                if(arr1[i] !== arr2[i]) {
+                    mismatch ++;
```

Comment:
If this is found, you could return `false` instead of breaking, achieves the same goal